tests/calculations/functions/test_create_kpoints_from_distance.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+"""Tests for the `create_kpoints_from_distance` calculation function."""
+from aiida.orm import Bool, Float
+
+from aiida_quantumespresso.calculations.functions.create_kpoints_from_distance import create_kpoints_from_distance
+
+
+def test_kpoints_00(generate_structure):
```

Comment:
I would give this test a more descriptive name, e.g.:

```suggestion
def test_3d_cubic(generate_structure):
```

That will make it easier to quickly parse which test failed when running `pytest`:

```
=========================================================================== short test summary info ===========================================================================
FAILED tests/calculations/functions/test_create_kpoints_from_distance.py::test_kpoints_00 - ValueError: this test broke
========================================================================= 1 failed, 2 passed in 0.86s =========================================================================

```

Or find the test you want to run in with the VSCode testing tool:

<img width="483" height="219" alt="Screenshot 2025-09-27 at 05 50 57" src="https://github.com/user-attachments/assets/7960525d-29fd-4c39-861c-c041bfeec744" />